examples/create_bids_folder.py

```diff
@@ -0,0 +1,54 @@
+"""
+==========================================
+Creating BIDS-compatible folders and files
+==========================================
+
+The Brain Imaging Data Structure (BIDS) has standard conventions for file
+names and folder hierarchy. MNE-BIDS comes with convenience functions if you
+wish to create these files/folders on your own.
+
+.. note::
+
+   You may automatically convert Raw files to a BIDS-compatible folder with
+   `raw_to_bids`. This example is for manually creating files/folders.
+"""
+
+# Authors: Chris Holdgraf <choldgraf@berkeley.edu>
+# License: BSD (3-clause)
+
+###############################################################################
+# We'll import the relevant functions from the utils module
+
+from mne_bids.utils import create_folders, filename_bids
```

Comment:
I have exposing modules called utils as name does not mean anything... 

if create_folders is a very common function then it should be in root module.